docs/src/main/asciidoc/security-oidc-code-flow-authentication-concept.adoc

```diff
@@ -267,285 +660,167 @@ quarkus.oidc.logout.post-logout-uri-param=returnTo
-----
-package io.quarkus.oidc.test;
-
-import jakarta.annotation.Priority;
-import jakarta.enterprise.context.ApplicationScoped;
-import jakarta.enterprise.inject.Alternative;
-import jakarta.inject.Inject;
-
-import io.quarkus.oidc.AuthorizationCodeTokens;
-import io.quarkus.oidc.OidcTenantConfig;
-import io.quarkus.oidc.TokenStateManager;
-import io.quarkus.oidc.runtime.DefaultTokenStateManager;
-import io.smallrye.mutiny.Uni;
-import io.vertx.ext.web.RoutingContext;
-
-@ApplicationScoped
-@Alternative
-@Priority(1)
-public class CustomTokenStateManager implements TokenStateManager {
-
-    @Inject
-    DefaultTokenStateManager tokenStateManager;
-
-    @Override
-    public Uni<String> createTokenState(RoutingContext routingContext, OidcTenantConfig oidcConfig,
-            AuthorizationCodeTokens sessionContent, TokenStateManager.CreateTokenStateRequestContext requestContext) {
-        return tokenStateManager.createTokenState(routingContext, oidcConfig, sessionContent, requestContext)
-                .map(t -> (t + "|custom"));
-    }
-
-    @Override
-    public Uni<AuthorizationCodeTokens> getTokens(RoutingContext routingContext, OidcTenantConfig oidcConfig,
-            String tokenState, TokenStateManager.GetTokensRequestContext requestContext) {
-        if (!tokenState.endsWith("|custom")) {
-            throw new IllegalStateException();
-        }
-        String defaultState = tokenState.substring(0, tokenState.length() - 7);
-        return tokenStateManager.getTokens(routingContext, oidcConfig, defaultState, requestContext);
-    }
+quarkus.oidc.auth-server-url=http://localhost:8180/realms/quarkus
+quarkus.oidc.client-id=frontend
+quarkus.oidc.credentials.secret=secret
+quarkus.oidc.application-type=web-app
 
-    @Override
-    public Uni<Void> deleteTokens(RoutingContext routingContext, OidcTenantConfig oidcConfig, String tokenState,
-            TokenStateManager.DeleteTokensRequestContext requestContext) {
-        if (!tokenState.endsWith("|custom")) {
-            throw new IllegalStateException();
-        }
-        String defaultState = tokenState.substring(0, tokenState.length() - 7);
-        return tokenStateManager.deleteTokens(routingContext, oidcConfig, defaultState, requestContext);
-    }
-}
+quarkus.oidc.logout.backchannel.path=/back-channel-logout
 ----
 
-=== Proof key for code exchange (PKCE)
+Absolute `Back-Channel Logout` URL is calculated by adding `quarkus.oidc.back-channel-logout.path` to the current endpoint URL, for example, `http://localhost:8080/back-channel-logout`. You will need to configure this URL in the Admin Console of your OpenID Connect Provider.
+
+Note that you will also need to configure a token age property for the logout token verification to succeed if your OpenID Connect Provider does not set an expiry claim in the current logout token, for example, `quarkus.oidc.token.age=10S` sets a number of seconds that must not elapse since the logout token's `iat` (issued at) time to 10.
```

Comment:
```suggestion
You will also need to configure a token age property for the logout token verification to succeed if your OpenID Connect Provider does not set an expiry claim in the current logout token.
For example, set `quarkus.oidc.token.age=10S` to ensure that no more than 10 seconds elapse since the logout token's `iat` (issued at) time.
```